pkg/secrets/clients/conjur/conjur_client.go

```diff
@@ -3,7 +3,8 @@ package conjur
 import (
 	"github.com/cyberark/conjur-api-go/conjurapi"
 
-	"github.com/cyberark/secrets-provider-for-k8s/pkg/log"
+	"github.com/cyberark/conjur-authn-k8s-client/pkg/log"
```

Comment:
this should go with the import above. 

the format is
```
import(
  <Go libraries>

  <External github libraries>

  <this github library>
)
```

Reply:
done